go/apps/ctrl/services/build/backend/docker/create_build.go

```diff
@@ -0,0 +1,149 @@
+package docker
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"connectrpc.com/connect"
+	"github.com/docker/docker/api/types/build"
+	"github.com/docker/docker/client"
+	ctrlv1 "github.com/unkeyed/unkey/go/gen/proto/ctrl/v1"
+	"github.com/unkeyed/unkey/go/pkg/assert"
+)
+
+type dockerBuildResponse struct {
+	Stream      string `json:"stream,omitempty"`
+	Error       string `json:"error,omitempty"`
+	ErrorDetail struct {
+		Code    int    `json:"code"`
+		Message string `json:"message"`
+	} `json:"errorDetail"`
+}
+
+func (d *Docker) CreateBuild(
+	ctx context.Context,
+	req *connect.Request[ctrlv1.CreateBuildRequest],
+) (*connect.Response[ctrlv1.CreateBuildResponse], error) {
+	if err := assert.All(
+		assert.NotEmpty(req.Msg.BuildContextPath, "build_context_path is required"),
+		assert.NotEmpty(req.Msg.UnkeyProjectId, "unkey_project_id is required"),
+	); err != nil {
+		return nil, connect.NewError(connect.CodeInvalidArgument, err)
+	}
+
+	const architecture = "amd64"
+	platform := fmt.Sprintf("linux/%s", architecture)
```

Comment:
this shouldn;'t be hardcoded and if you rename the depot_platform to a generic var we can just pass it into the docker builder too. 

Since even locally one might want to build the architecture they have running e.g arch/amd

Reply:
Yeah I forgot to replace this one with new env. I'll replace these too